kernel/src/sys/proc.c

```diff
@@ -44,7 +44,7 @@ process_t* proc_run_code(uint8_t* code, uint32_t size, char** argv) {
         argv++;
     }
 
-    // Allocate one page more than the program size to accomodate static
+    // Allocate one page more than the program size to accommodate static
     // variables.
```

Comment:
Since I always spell things wrong (and it gets worse due the fact English is not my native language) I installed a VS Code extension that checks grammar for me